content/nginx-one/staged-configs/api-staged-config.md

```diff
@@ -0,0 +1,20 @@
+---
+# We use sentence case and present imperative tone
+title: Use the API to manage your Staged Configurations
+# Weights are assigned in increments of 100: determines sorting order
+weight: 300
+# Creates a table of contents and sidebar, useful for large documents
+toc: true
+# Types have a 1:1 relationship with Hugo archetypes, so you shouldn't need to change this
+type: tutorial
+# Intended for internal catalogue and search, case sensitive:
+product: NGINX One
+---
+
+You can use F5 NGINX One Console API to manage your Staged Configurations. With our API, you can:
```

Comment:
```suggestion
Use F5 NGINX One Console API to manage your Staged Configurations. With our API, you can:
```

Reply:
MSTP says to delete can if possible. Instead of "You can" just use an imperative.

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/can-may